mw-webapp/src/dataAccessLogic/UserPreviewDAL.ts

```diff
@@ -13,15 +13,15 @@ export class UserPreviewDAL {
   /**
    * Get amount of all users in collection
    */
-  public static async getUsersPreviewAmount(): Promise<number> {
-    return await UserService.getUsersDTOAmount();
+  public static async getUsersPreviewAmount(filterEmail?: string): Promise<number> {
```

Comment:
use open close pronciple to write right arguments
(filter: {email?: string})

Reply:
Changed